theano/gof/cmodule.py

```diff
@@ -2105,10 +2261,23 @@ def compile_str(module_name, src_code, location=None,
         if python_lib not in lib_dirs:
             lib_dirs.append(python_lib)
 
-        cppfilename = os.path.join(location, 'mod.cpp')
-        cppfile = open(cppfilename, 'w')
+        if shared:
+            hide_symbols = False
```

Comment:
I think that for now, this is good. But can you add a todo to tell that we should explicitly mark as visible the part that need that and don't override that?